xla/service/gpu/transforms/priority_fusion.cc

```diff
@@ -352,6 +353,48 @@ class PriorityFusionQueue {
     fusion_info_cache_.Invalidate(instruction);
   }
 
+  void UpdateRuntimes(
+      GpuPerformanceModel::RunTimes& runtimes, const HloInstruction* consumer,
+      const absl::flat_hash_map<const HloInstruction*, absl::Duration>&
+          original_consumers) {
+    auto it = original_consumers.find(consumer);
+    if (it != original_consumers.end()) {
```

Comment:
This can probably be `if (original_consumers.contains(consumer)) { ... }`.

Reply:
I think we need the iterator here to get the time_fused.